### PR TITLE
Implemented Anonymize _ssn. Updated _ssn test cases

### PR DESCRIPTION
**Pull request by jpalm60**

### CODE REVIEW

**Review by abelcano — APPROVED** [review]

File: test_pii_data.py [hunk 1]
```diff
@@ -138,6 +138,36 @@ def test_has_at_handle(self):
...
+        test_data = Pii('My social security is 098.67.9878')
+        self.assertFalse(test_data.has_ssn())
+        test_data = Pii('My social security is 098679878')
+        self.assertFalse(test_data.has_ssn())
+        test_data = Pii('My social security is 098-6-9878')
+        self.assertFalse(test_data.has_ssn())
+
+    def test_has_ssn_anonymize(self):
+        test_data = Pii('My social security is 123-45-5667')
+        self.assertEqual(test_data.has_ssn(anonymize=True), 'My social security is [social security number]')
```
Comment (claesmk) [inline test_pii_data.py:159]: Change this to [ssn number]

File: test_pii_data.py [hunk 2]
```diff
@@ -138,6 +138,36 @@ def test_has_at_handle(self):
...
+        test_data = Pii('My social security is 654-45-3456')
+        self.assertEqual(test_data.has_ssn(anonymize=True), 'My social security is [social security number]')
```
Comment (claesmk) [inline test_pii_data.py:161]: Change this to [ssn number]

**Review by claesmk — CHANGES_REQUESTED** [review]
Please change [ssn number] as noted in comments

**Comment by jpalm60** [comment]
The Issues have been resolved

**Review by claesmk — APPROVED** [review]
Looks good

**Pull request merged**